webview-ui/src/components/ViolationIncidentsList.tsx

```diff
@@ -183,12 +249,22 @@ const ViolationIncidentsList = ({
       groups.get(key)!.incidents.push(incident);
     });
 
-    return Array.from(groups.entries()).map(([id, { label, incidents }]) => ({
+    const sortedGroups = Array.from(groups.entries()).map(([id, { label, incidents }]) => ({
       id,
       label,
       incidents,
     }));
-  }, [enhancedIncidents, searchTerm, filters]);
+
+    if (sortField !== "none") {
+      sortedGroups.sort((a, b) => {
+        const fieldA = a.label.toLowerCase();
+        const fieldB = b.label.toLowerCase();
```

Comment:
You're not using the `sortField`.  I see the label being set in the switch above based on the grouping, but that leads to a different question.

Why have sorting on file name when the group by is incident and why have sorting on incident when the group by is file?

Sorting would be much more natural if the webview was an actual table instead of a collection of cards.